python/lib/solver.cc

```diff
@@ -678,11 +827,46 @@ WrapPIteration(py::module& slv)
       }
     ),
     R"(
-    Construct a power iteration k-eigen SCDSA solver object.
+    Construct a power iteration k-eigenvalue solver with SCDSA.
 
     Parameters
     ----------
-    ???
+    lbs_problem: LBSProblem
+        Existing LBSProblem instance.
+    max_iters: int, optional
+        Maximum power iterations allowed (default: 1000).
+    k_tol: float, optional
+        Tolerance on the k-eigenvalue (default: 1.0e-10).
+    reset_solution: bool, optional
+        If true, initialize flux moments to 1.0 (default: True).
+    reset_phi0: bool, optional
+        If true, reinitializes scalar fluxes to 1.0 (default: True).
+    accel_pi_max_its : int, optional, default=50
+        Maximum number of iterations allowed for the inner power iterations used by the
+        acceleration method.
+    accel_pi_k_tol : float, optional, default=1.0e-10
+        Convergence tolerance on the k-eigenvalue within the inner power iterations of the
+        acceleration method.
+    accel_pi_verbose : bool, optional, default=False
+        If True, enables verbose logging output from the acceleration method's inner solver.
+    diff_accel_diffusion_l_abs_tol : float, optional, default=1.0e-10
+        Absolute residual tolerance for convergence of the diffusion accelerator.
+    diff_accel_diffusion_max_iters : int, optional, default=100
+        Maximum number of iterations allowed for the diffusion accelerator solve.
+    diff_accel_diffusion_verbose : bool, optional, default=False
+        If True, enables verbose logging output from the diffusion accelerator.
+    diff_accel_diffusion_petsc_options : str, optional, default="ssss"
```

Comment:
"ssss" is really the default?

Reply:
That is the current default. Probably just a placeholder. I'll address this in a separate PR and update the binding documentation.

Reply:
👍